standard/expressions.md

```diff
@@ -687,7 +690,7 @@ When arguments are omitted from a function member with corresponding optional pa
 
 When a generic method is called without specifying type arguments, a ***type inference*** process attempts to infer type arguments for the call. The presence of type inference allows a more convenient syntax to be used for calling a generic method, and allows the programmer to avoid specifying redundant type information.
 
-> *Example*: Given the method declaration:
+> *Example*: Given the method declaration and use, it is possible to invoke the `Choose` method without explicitly specifying a type argument:
```

Comment:
Hmm... this doesn't quite read right to me, as "it is possible" suggests that it's possible to add more code to invoke the Choose method without explicitly specifying a type argument.

Is it worth just removing the sentence entirely, leaving just "Example:"? I can't remember whether every example has a description.

Reply:
OK; I removed the lead-in sentence.